static/app/views/performance/transactionDetails/content.tsx

```diff
@@ -162,6 +169,179 @@ class EventDetailsContent extends DeprecatedAsyncComponent<Props, State> {
 
     const originatingUrl = getUrlFromEvent(event);
 
+    const renderBody = (metaResults: TraceMetaQueryChildrenProps) => {
+      const {isLoading} = metaResults;
+
+      if (isLoading) {
+        return <LoadingIndicator />;
+      }
```

Comment:
This makes the page now block on the events-meta call so perhaps consider wrapping this in a feature flag check. And I think this appears in another place in your PR as well.

Reply:
Hey yup, for sure this PR is a bit everywhere. I walked through it with Nar who's reviewing it.  And yup this block definitely needs the feature flag check thanks!